server/bin/pbench-unpack-tarballs.py

```diff
@@ -43,6 +43,7 @@ def main(cfg_name):
 
     MIN_LOWERBOUND = 0.0
     MAX_UPPERBOUND = float("inf")
+    lowerbound, upperbound = 0.0, float("inf")
```

Comment:
You have those lovely symbolic constants...shouldn't you be using them?  😁 
```
    lowerbound, upperbound = MIN_LOWERBOUND, MAX_UPPERBOUND
```

Reply:
This was a good idea, but apparently was overlooked? 😆 Not necessarily a showstopper; but if you make other changes it'd be good to get this one.

Although, actually, since we don't need the `MAX_*` constants again after the `get` calls ... you could really *drop* the constants, simply assign the default `lowerbound` and `upperbound` values, and use *those* for `fallback=` values...